src/cloud-cli/meltano/cloud/cli/deployment.py

```diff
@@ -344,6 +346,23 @@ async def use_deployment(  # noqa: D103
     )
 
 
+def _set_project_default_deployment(
```

Comment:
This was reviewed by Will in a PR above this, I merged it into this since approved